Automattic/jetpack
# Fix reading of jetpack_verbum_subscription_modal

The value for `jetpack_verbum_subscription_modal` was improperly read for WoA atomic sites.
This PR fixes that.

Fixes #

## Proposed changes:

* Use ` get_blog_option( $this->blog_id,...)` ti ensure that the correct one is retrieved.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Sync this PR to your WoA site
* Using `_cli` for the site set the option with `wp option set jetpack_verbum_subscription_modal 0`
* Ensure that `VerbumComments.enableSubscriptionModal` reflects the correct value for the iframe
*
